geoips/image_utils/mpl_utils.py

```diff
@@ -220,7 +221,7 @@ def save_image(
             dpi=rc_params["figure.dpi"],
             pad_inches=0.0,
             transparent=True,
-            facecolor="none",
+            facecolor="white",
```

Comment:
We probably don't want to merge this as-is - this will change the default image output, correct?  Which will probably impact existing products / tests?